v1/worker.go

```diff
@@ -293,7 +293,7 @@ func (worker *Worker) keepAndRetryTaskIn(signature *tasks.Signature, retryIn tim
 	// only matter if the broker does not support RetryMessage and falls back to sending a new task.
 	signature.AttemptCount++
 
-	log.WARNING.Printf("Task %s failed. Going to retry in %.0f seconds. Attempting to keep message.", signature.UUID, retryIn.Seconds())
+	log.WARNING.Printf("Task %s (%s) failed. Going to retry in %.0f seconds, attempt count %d. Attempting to keep message.", signature.UUID, signature.Name, signature.AttemptCount, retryIn.Seconds())
```

Comment:
I think the last 2 params are out of order here.

Reply:
Thanks! I'll address!